src/components/global/page-header.tsx

```diff
@@ -15,7 +15,7 @@ const PageHeader = async () => {
       <GlobalMessage />
       <div className="min-h-50 relative z-[2] border-b border-fog">
         <div className="grow-1 centered flex items-center justify-between gap-20 pr-24 lg:pr-0">
-          <div className="rs-pt-0 rs-pb-1 w-full lg:rs-pt-1 lg:rs-pb-2">
+          <div className="rs-pt-0 rs-pb-1 lg:rs-pt-1 lg:rs-pb-2">
```

Comment:
Taking this `w-full` out cause while it resolves the Safari issue, it's causing the Header SVG to appeared center to the parent container size. We want the Logo to be left aligned with our body content. 

I quickly tested out some aspect ratio things (e.g. aspect-[705/84] but that seems to be another rabbit hole. So I'm going to time box it again and push up a seperate PR later today. 🤔 We might just need to go back to fixed heights at certain media queries or container queries (container queries might be a great option for this one actually).